addons/test_mail/tests/test_mail_followers.py

```diff
@@ -147,6 +147,17 @@ def test_followers_default_partner_context(self):
             channel_ids=[self.channel_listen.id]
         )
 
+    def test_recipient_data_group_no_subtype(self):
```

Comment:
Could be great to run as a real user (like @users('employee'), always a better idea than relying on admin, even for technical tests)